backend/src/main/webapp/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Is that possible to move this to the root `.gitignore` file?

Reply:
It's possible, but I'm not sure if it would make sense in the context of the whole app (maybe) as this is mostly for the UI. I'll check.

Reply:
From my experience, it's easier to manage a single `.gitignore` file at root level in a multi-modules Maven project rather than multiple files. But I don't have a strong opinion on that, I can live with multiple files 😃

Reply:
I think it's a little awkward to mix them together, but can be  done:

from:
```
# dependencies
/node_modules
/.pnp
.pnp.js

```

to (1):

```
## dependencies
/backend/src/main/webapp/node_modules
/backend/src/main/webapp/.pnp
/backend/src/main/webapp/**/.pnp.js
```

or (2)

```
node_modules
.pnp
.pnp.js
```

(1) is more exact on what we want, while (2) is less awkward, different to the main intent but similar. 
WDYT?